frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-card.tsx

```diff
@@ -175,12 +176,26 @@ export const PrometheusUtilizationItem = withDashboardResources<PrometheusUtiliz
     let request: PrometheusResponse, requestError: any;
     let isLoading = false;
 
+    const { t } = useTranslation();
+
+    const UTILIZATION_QUERY_DURATION = {
```

Comment:
cant we use the `Duration` defined in the `duration-hook.ts` ? so we dont repeat ourselves